queue/util/generateDeadLetterExchangeName.js

```diff
@@ -0,0 +1,6 @@
+/**
+* Generates a dead letter exchange name from a queueName
+*/
+module.exports = function (queueName) {
+  return `${queueName}-dead-letter-exchange`
+}
```

Comment:
Also super minor, but may as well use named functions in places like this for easier debugging.